mlflow/tracking/fluent.py

```diff
@@ -347,8 +539,30 @@ def get_experiment_by_name(name):
     """
     Retrieve an experiment by experiment name from the backend store
 
-    :param name: The experiment name.
+    :param name: The case senstive experiment name.
     :return: :py:class:`mlflow.entities.Experiment`
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Case sensitive name
+        experiment = mlflow.get_experiment_by_name("Default")
+
+        # Print the contents of Experiment data
+        print("Experiment_id={}".format(experiment.experiment_id))
```

Comment:
```suggestion
        print("Experiment_id: {}".format(experiment.experiment_id))
```

This is my personal pereference, but if you print something with the `key=value` format, it's a bit unclear where the value starts.

Reply:
Looking back, I think I like this format. Will change the printed out values in `key:value` format.

Reply:
Consider it done!